test/Microsoft.TemplateEngine.IDE.IntegrationTests/Utils/BasicParametersParser.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.TemplateEngine.IDE.IntegrationTests.Utils
+{
+    internal static class BasicParametersParser
+    {
+        internal static string GetNameFromParameterString(string parameters)
+        {
+            string[] parametersArray = parameters.Split(null);
+
+            int nameIndex = Array.IndexOf(parametersArray, "--name");
+            if (nameIndex >= 0 && nameIndex + 1 < parameters.Length)
```

Comment:
Index zero is included in this method, but not in the one below. Is this intended? If yes, should we add a comment to remind us?